aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -130,9 +130,9 @@ MPSDataType getMPSScalarType(ScalarType scalar_type) {
     case ScalarType::Short:
       return "Int16";
     case ScalarType::Char:
-      return "UInt8";
-    case ScalarType::Byte:
       return "Int8";
+    case ScalarType::Byte:
```

Comment:
@razarmehr , did you see this issue as well, in some of the uint8 test cases? it seems like a good fix.